README.md

```diff
@@ -1,101 +1,91 @@
+
 # The Amazon Kinesis Video WebRTC Sample
+[![Build Status](https://travis-ci.org/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios.svg?branch=master)](https://travis-ci.org/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios)
+[![Coverage Status](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/branch/master/graph/badge.svg)](https://codecov.io/gh/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios)
 
-This sample demonstrates the Amazon Cognito Identity Provider found in the AWS Mobile SDK for iOS.
+This sample demonstrates the Amazon KinesisVideoStreams and KinesisVideoSignaling framework found in the AWS Mobile SDK for iOS.
 
 ## Requirements
 
-* Xcode 9.2 and later
+* Xcode 10 and later
 * iOS 11 and later
 
 #### Download the WebRTC SDK in iOS
 To download the WebRTC SDK in iOS, run the following command:
 
-	git clone https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios.git
+    git clone https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios.git
   
 
 #### Using XCode to build the project
 
 1. The AWS Mobile SDK for iOS is available through [CocoaPods](http://cocoapods.org). If you have not installed CocoaPods, install CocoaPods: (Requires Ruby installed) 
 
-		sudo gem install cocoapods
-		pod setup
+        sudo gem install cocoapods
+        pod setup
 
 2. To install the AWS Mobile SDK for iOS run the following command in the directory containing this sample:
 
-		pod install
+        pod install
 
 3. Create an Amazon Cognito User Pool. Follow the 4 steps under **Creating your Cognito Identity user pool** in this [blog post](http://mobile.awsblog.com/post/TxGNH1AUKDRZDH/Announcing-Your-User-Pools-in-Amazon-Cognito).
 
 4. Open `KinesisVideoWebRTCDemoApp.xcworkspace` (File location: KVSiOS/Swift/AWSKinesisVideoWebRTCDemoApp.xcworkspace).
 
 5. Open **Constants.swift**. Set **CognitoIdentityUserPoolRegion**, **CognitoIdentityUserPoolId**, **CognitoIdentityUserPoolAppClientId** and **CognitoIdentityUserPoolAppClientSecret** to the values obtained when you created your user pool.
 ```swift
-		let CognitoIdentityUserPoolRegion: AWSRegionType = .Unknown
-		let CognitoIdentityUserPoolId = "YOUR_USER_POOL_ID"
-		let CognitoIdentityUserPoolAppClientId = "YOUR_APP_CLIENT_ID"
-		let CognitoIdentityUserPoolAppClientSecret = "YOUR_APP_CLIENT_SECRET"
+        let CognitoIdentityUserPoolRegion: AWSRegionType = .Unknown
+        let CognitoIdentityUserPoolId = "YOUR_USER_POOL_ID"
+        let CognitoIdentityUserPoolAppClientId = "YOUR_APP_CLIENT_ID"
+        let CognitoIdentityUserPoolAppClientSecret = "YOUR_APP_CLIENT_SECRET"
 ```
 Replace the “REPLACEME” places in each of these files with the appropriate AWS Credentials:
   *  _awsconfiguration.json_ (path: KVSiOS/Swift/KVSiOSApp/awsconfiguration.json)
-  *	 _Constants.swift_ (path: KVSiOS/Swift/KVSiOSApp/Constants.swift)
-
+  *     _Constants.swift_ (path: KVSiOS/Swift/KVSiOSApp/Constants.swift)
```

Comment:
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/blob/master/Swift/KVSiOSApp/Constants.swift might provide less confusion comparing to "KVSiOS" which is not mentioned anywhere.